### PR TITLE
[AppSec] Remove instances of Build Integrity from Classic

### PR DESCRIPTION
**Pull request by anaghapamidi**
Modified Files:

- docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc
- docs/en/classic/appsec-admin-guide/risk-prevention/code/monitor-fix-issues-in-scan.adoc

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc [hunk 1]
```diff
@@ -50,54 +47,54 @@ To understand the default scan parameter on Prisma Cloud with the enforcement ru
...
 
+** *Code Reviews*: The *Code Reviews* view provides a real-time monitoring results, offering contextual information about code security across repositories. Utilize graphical representation to comprehend the overall distribution of Failed and Passed scans. You can directly inspect the scan on the version control system (VCS). Further, you can delve into the specifics of the latest scan results and access supplementary details on *Application Security > Projects*.
+
+By combining these two views, you can interpret the scan results, ensuring that you have a comprehensive approach for a quick identification of potential vulnerabilities and facilitate remediation and improvement.
+//Currently the ability to review violation fix suggestions and view the Pull Request (PR) scans that failed is supported only for Github repositories. From the Prisma Cloud console you can directly access your repositories in Github and remediate solutions through a Pull Request (PR).
```
Comment (anaghapamidi) [inline docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc:97]: @@tsmithv11 - Does this statement still hold true?

File: docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc [hunk 2]
```diff
@@ -228,41 +222,36 @@ You can set a run rule off for a code category in either a default configuration
 
 [.task]
 
-[#review-fail-scans-and-suggestions-on-vcs]
-=== Review fail scans and suggestions on VCS (Version Control System)
+[#review-scan-result-and-suggestions-for-vcs]
+=== Review scan results and suggestions for VCS (Version Control System)
```
Comment (anaghapamidi) [inline docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc:226]: @@tsmithv11 - Can I please request a review for this section? This is for the Classic UI and had some inconsistencies.

File: docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc [hunk 3]
```diff
@@ -228,41 +222,36 @@ You can set a run rule off for a code category in either a default configuration
...
 
-After a scan result that fails the enforcement configuration, to find remediation you can directly access your the latest Pull Request (PR)  from the Enforcement scan result.
+To thoroughly examine the scan results of your repositories following the execution of an Enforcement configuration in the Prisma Cloud console, see the steps here for seamless and insightful review.
```
Comment (tsmithv11) [inline docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc:228]:
```suggestion
To examine the results of your repository scans with the Enforcement configuration in the Prisma Cloud console, see the steps here for seamless and insightful review.
```

File: docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc [hunk 4]
```diff
@@ -228,41 +222,36 @@ You can set a run rule off for a code category in either a default configuration
...
+.. Select *Application Security > Development Pipelines*.
 +
-image::enfor-14.png[width=800]
+image::enfor-14.png[]
 
-. Select *Actions* corresponding to the fail scan result.
+. Projects View
 +
-image::enfor-23.png[width=600]
+By default, you will see the *Projects* view, offering a comprehensive overview of your repositories. This view includes crucial information, such as *Failed PR*, *Pending PR*, and the *Latest PR*. This snapshot is enriched with data on *Weekly commits* and the number of *Git users* interacting with the repository.
```
Comment (tsmithv11) [inline docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc:239]:
```suggestion
By default, you will see the *Projects* view, offering a helpful overview of your repositories and the pull request activity. This view includes crucial information, such as *Failed PR*, *Pending PR*, and the *Latest PR*. This snapshot is enriched with data on *Weekly commits* and the number of *Git users* interacting with the repository.
```

File: docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc [hunk 5]
```diff
@@ -228,41 +222,36 @@ You can set a run rule off for a code category in either a default configuration
...
-NOTE: Ensure you have access to the repository on Github.
+..Select *Open the latest scan item* to access more detailed information about the repository directing you to *Application Security > Projects*.
 
-. Select *Open failed PRs scans* to view a list of Pull Request (PR) that have failed with your repository on Github.
-+
-image::enfor-17.png[width=800]
-+
-You can choose to remediate the repository on Github.
+. Code Reviews View
+.. Select *Code Reviews* to gain real-time insights into the security status of repositories post-Enforcement configuration. Here, you have two key *Actions*:
```
Comment (tsmithv11) [inline docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc:254]:
```suggestion
.. Select *Code Reviews* to gain real-time insights into the security status of repository scans with Enforcement configurations applied. Here, you have two key *Actions*:
```

**Pull request merged**